docs/gallery/plot_external_resources.py

```diff
@@ -320,3 +323,34 @@ def __init__(self, **kwargs):
 
 er_read = ExternalResources.from_norm_tsv(path='./')
 os.remove('./er.zip')
+
+###############################################################################
+# Using TermSet with ExternalResources
+# ------------------------------------------------
+# :py:class:`~hdmf.TermSet` allows for an easier way to add references to
+# :py:class:`~hdmf.common.resources.ExternalResources`. These enumerations take place of the
+# entity_id and entity_uri parameters. :py:class:`~hdmf.common.resources.Key` values will have
+# to match the name of the term in the :py:class:`~hdmf.TermSet`.
+import linkml_runtime
```

Comment:
```suggestion
import linkml_runtime
```
I guess this import is not needed anymore.